src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Review.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@using SFA.DAS.Encoding
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort.ReviewViewModel
+@inject IEncodingService EncodingService
+
+@{
+    ViewData["Title"] = "Review";
+}
+
+<div class="govuk-grid-row">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-width-container">
+            <h1 class="govuk-heading-xl">Apprentice details ready for review</h1>
+
+            <table class="govuk-table das-table--responsive">
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header das-table-cell-width-25">Training provider</th>
+                        <th scope="col" class="govuk-table__header das-table-cell-width-10">Reference</th>
+                        <th scope="col" class="govuk-table__header das-table-cell-width-25 govuk-table__header--numeric">Number of apprentices</th>
+                        <th scope="col" class="govuk-table__header das-table-cell-width-25">Last Message</th>
+                        <th scope="col" class="govuk-table__header govuk-table__header--numeric das-table-cell-width-15"></th>
+                    </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+
+                    @foreach (var cohort in Model.CohortSummary)
+                    {
+
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell" data-label="Training provider">@cohort.ProviderName</td>
+                        <td class="govuk-table__cell" data-label="Reference">@cohort.CohortReference</td>
+                        <td class="govuk-table__cell govuk-table__cell--numeric" data-label="Number of apprentices">@cohort.NumberOfApprentices</td>
+                        <td class="govuk-table__cell" data-label="Last Message">@cohort.LastMessage</td>
+                        <td class="govuk-table__cell govuk-table__cell--numeric" data-label="">
+                            @Html.ActionLink("Details", "Details", "Cohort", new { @Model.AccountHashedId, cohort.CohortReference }, new { @class = "govuk-link", id = "return-link" })
```

Comment:
As on Draft page, having a repeating Id should be avoided and the proposed Id is wrong. We could just add a class to help automation testing.

Reply:
Agreed, the Id shouldn't repeat, ( copy&paste gone wrong :). As suggested I created a class.